src/connect-mixin.ts

```diff
@@ -35,15 +37,15 @@ export const connect = (store) => (baseElement) => class extends baseElement {
   }
 
   disconnectedCallback() {
-    this.__storeUnsubscribe();
+    this.__storeUnsubscribe!();
 
     if (super.disconnectedCallback) {
       super.disconnectedCallback();
     }
   }
 
   // This is called every time something is updated in the store.
-  _stateChanged(state) {
-    throw new Error('_stateChanged() not implemented', this);
+  _stateChanged(_state: S) {
```

Comment:
This looks `protected`

Reply:
looks like this is a TS no-go: https://github.com/Microsoft/TypeScript/issues/17293